pyproject.toml

```diff
@@ -35,6 +35,7 @@ autoapi = [
     "sphinx-autoapi==3.4.0",
     "sphinx-design==0.6.1",
     "sphinx-jinja==2.0.2",
+    "astroid>=3.3.8,<4.0",
```

Comment:
Pretty restrictive though, right? Why not above 3.3 or even 3?

Reply:
sure, added 3.

Reply:
Awesome, all good now!